### PR TITLE
Add READMEs

### PR DESCRIPTION
**Pull request by annalisev**
I have added READMEs for:
notification
database
terraform

### CODE REVIEW

**Comment by annalisev** [comment]
Closes #205 
Closes #206
Closes #210

**Comment by annalisev** [comment]
Closes #212

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Description
 
-> This Project is responsible for helping users to keep up with the latest releases as well as understand latest gaming trends.
+> This Project is responsible for helping users to keep up with the latest releases as well as understand the latest gaming trends.
```
[the remark this answers is not in the document]
Comment (annalisev) [reply README.md:3]: Can be

**Comment by annalisev** [comment]
Closes #213

File: README.md (hunk 1 above)
Comment (annalisev) [inline README.md:3]:
```suggestion
> This Project is responsible for helping users to keep up with the latest releases as well as understand latest gaming trends.
```

File: database/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+## Description
+
+This Folder is responsible for defining and creating the tables in our database.
+
+
+## Requirements
+There are no requirements for this folder.
```
Comment (hhm970) [inline database/README.md:7]: Include `brew install postgresql@14` as a requirement

File: terraform/README.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
+## Description
+
+This Folder contains all the terraform files to create all our AWS services.
+
+
+## Requirements
```
Comment (hhm970) [inline terraform/README.md:6]: Make sure to include 'Run `brew install terraform`' as an installation step

File: database/README.md (hunk 2 above)
Comment (annalisev) [reply database/README.md:7]: I think Shindy is covering this in the main README

File: terraform/README.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+- STEAM_BASE_URL
+- AWS_KEY
+- AWS_SECRET
+
+## The Scripts
+This folder contains the following scripts:
+
+- **database.tf**  
+This script creates an RDS database.  
+
```
Comment (hhm970) [inline terraform/README.md:28]:
```suggestion
  - **dashboard.tf**
  This script creates an ECS Service which continually deploys the dashboard.
  
```

Comment (hhm970) [reply terraform/README.md:28]: ^^ I'm aware this may not be perfect, so feel free to write your own!

File: pipeline/notification/README.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+This script sends SNS alerts to subscribers of topics, based on newly released games.  
+
+   Run from the command line using: 
+  >python3 alert.py
+  
+- **Dockerfile**  
+This script is used to dockerise **alert.py**.  
+  
+  Run from the command line using: 
+  >docker build -t alert .
```
Comment (hhm970) [inline pipeline/notification/README.md:33]:
```suggestion
  >docker build -t alert --platform "linux/amd64" .  
```

Comment (hhm970) [reply pipeline/notification/README.md:33]: Just realised this says 'run from command line' so feel free to ignore my suggestion!

Comment (annalisev) [reply pipeline/notification/README.md:33]: I was thinking maybe we don't need this when running locally?

**Review by hhm970 — APPROVED** [review]
The whole README.md seems very thorough and coherent - well done!

**Pull request merged**